### ISSUE TITLE
Show actual values

### ISSUE THREAD

**Issue by Turbo87** [F-Secure/pytest-voluptuous#3]
```
E       AssertionError: assert failed to validation error(s):
E         - results.flight.id: not a valid value for dictionary value @ data['results']['flight']['id']
```

It's nice that it shows me what values specifically do not comply with the schema, but it would be awesome if it could also show the actual value so that failing tests are easier to debug.

**Comment by mtvx** [comment]
Released in 1.1.0. Thanks!

**Comment by Turbo87** [comment]
sweet, thanks! :)
